bip-0340/test-vectors.py

```diff
@@ -1,6 +1,16 @@
 import sys
 from reference import *
 
+def is_square(x):
+    return int(pow(x, (p - 1) // 2, p)) == 1
+
+def has_square_y(P):
+    """Determine if P has a square Y coordinate. Used in an earlier draft of BIP340."""
+    infinity = is_infinity(P)
+    if infinity: return False
+    assert P is not None
```

Comment:
without the mypy types this looks very weird. Perhaps just
```
"""<comment>"""
assert not is_infinity(P)
return is_square(P[1])

Reply:
Done.